### PR TITLE
Update about.md

### PR DESCRIPTION
**Pull request by snyder114**
Minor readability enhancements

### CODE REVIEW

File: content/en/docs/collector/about.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+Weight: 1
 ---
 
 ## Introduction
 
-The OpenTelemetry Collector offers a vendor-agnostic implementation on how to
+The OpenTelemetry Collector offers a vendor-agnostic way to
 receive, process, and export telemetry data. It removes the need to run,
-operate, and maintain multiple agents/collectors in order to support
+operate, and maintain multiple agents or collectors in order to support
```
Comment (flands) [inline content/en/docs/collector/about.md:10]:
```suggestion
operate, and maintain multiple agents or gateways in order to support
```

File: content/en/docs/collector/about.md [hunk 2]
```diff
@@ -125,16 +125,16 @@ $ git clone git@github.com:open-telemetry/opentelemetry-collector.git; \
 
 ## Other
 
-The OpenTelemetry collector can be extended or embedded into other applications.
+You can extend the OpenTelemetry collector or embedded it into other applications.
```
Comment (flands) [inline content/en/docs/collector/about.md:128]:
```suggestion
You can extend the OpenTelemetry Collector or embed it into other applications.
```

**Review by flands — COMMENTED** [review]
LGTM, few comments

**Comment by flands** [comment]
@snyder114 can you fix the conflict?

**Comment by snyder114** [comment]
> @snyder114 can you fix the conflict?

@flands I only have read access to the repo :( Would you mind either fixing the conflict or somehow giving me write access? Is that possible?

**Comment by snyder114** [comment]
@flands any updates on this? Should i just close the PR and open a new one?

**Comment by austinlparker** [comment]
Could you close and reopen after pulling in changes? It won't let me edit the conflicts directly.

**Comment by flands** [comment]
sorry for the delay -- unfortunately we cannot edit it so please close and reopen

**Comment by austinlparker** [comment]
please rebase locally and re-open this PR if you wish to contribute it. sorry!

**Pull request closed without merging**